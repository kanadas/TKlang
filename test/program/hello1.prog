stream main
input Input
state
output print = 'h'
initial
