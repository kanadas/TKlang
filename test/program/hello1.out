Stream: main has uninitialized values
